check_stability.py

```diff
@@ -384,12 +384,32 @@ def install_wptrunner():
     call("pip", "install", wptrunner_root)
 
 
-def get_files_changed():
+def get_branch_point(user):
+    git = get_git_cmd(wpt_root)
+    if os.environ.get("TRAVIS_PULL_REQUEST", "false") != "false":
```

Comment:
I think `"TRAVIS_PULL_REQUEST" in os.environ` would communicate your intention more clearly.